src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.dto.mapping.impl.response.OrderResponseMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
+@RequiredArgsConstructor
+@RequestMapping("/orders")
+public class OrderController {
+    OrderService orderService;
+    UserService userService;
+    ShoppingCartService shoppingCartService;
+    OrderResponseMapper orderResponseMapper;
+
+    @GetMapping
+    public List<OrderResponseDto> getOrderHistoryByUser(@RequestParam Long userId) {
+        var user = userService.get(userId);
```

Comment:
it seems to me that var should be used if the type of the variable is very long (`List<Map<String, Map<String, SomeVeryVeryLongClassName>>>  someVariable` or smth like this), in this case I don’t see the benefit of var usage...